drivers/scheduler/k8s/specs/fio-io-throttle/fio-config-map.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fio-job-config
+data:
+  fio.job: |
+    [global]
+    name=fio-rand-RW
+    filename=/scratch/fio.dat
+    rw=write
```

Comment:
readwrite to test both read and write

Reply:
you can do readwrite but then you have to look for both bandwidths. I would say keep them separate.